### DISCUSSION TITLE
2024年6月6日续费付款2年 专业版 但没有激活码

### DISCUSSION THREAD

**Discussion by kennylo47** [getlantern/lantern-forum-cn#602] [Q&A]
![WechatIMG868](https://github.com/getlantern/lantern-forum-cn/assets/173150461/8625a03d-ffca-4ef2-8e6d-3f0d75c890c3)
邮箱lawy2k47@hotmail.com
麻烦发激活码到这里
现在升级到7.8.8 都没办法用专业版
谢谢！

**Comment by skytony** [comment]
可能大家都上当了，我也是同样的问题...

**Reply by MaxMKMax** [reply]
@kennylo47 谢谢您支持蓝灯. 好的, 我帮您提交问题. 请您过几天查看这个帖子或邮箱. 我们会给您发激活码.

**Reply by MaxMKMax** [reply]
@skytony 
您好. 我们在市场上超过十年了. 有品牌保证的. 如果您也遇到同样的问题, 我帮您处理. 请您提供我支付单据和账号邮箱.
谢谢您支持蓝灯. 我帮您提交问题. 请您过几天查看这个帖子或邮箱. 我们会给您发激活码.

**Comment by kennylo47** [comment]
邮箱收到mike的邮件回复说已经处理好了，但我没有看到有激活码啊？lantern依旧是普通状态

**Comment by kennylo47** [comment]
过去3天了 依然没有任何解决实际的东西。到底是行还是不行？ 不行就退费了 @Mike777Khoo

**Reply by MaxMKMax** [reply]
@kennylo47 您好. 我再重新帮您问一下. 尽快给您答复. 不好意思让您久等了.

**Comment by MaxMKMax** [comment]
@kennylo47 帮您提交问题了, 这两天内给您答复和解决激活码的问题. 谢谢!

**Comment by kennylo47** [comment]
已经解决了 感谢

获取 Outlook for iOS<https://aka.ms/o0ukef>
________________________________
发件人: Mike K ***@***.***>
发送时间: Monday, July 29, 2024 11:36:36 PM
收件人: getlantern/lantern-forum-cn ***@***.***>
抄送: kennylo47 ***@***.***>; Mention ***@***.***>
主题: Re: [getlantern/lantern-forum-cn] 2024年6月6日续费付款2年 专业版 但没有激活码 (Discussion #602)


@kennylo47<https://github.com/kennylo47> 帮您提交问题了, 这两天内给您答复和解决激活码的问题. 谢谢!

―
